### ISSUE TITLE
Import/Export User list

### ISSUE THREAD

**Issue by RaziaMahmodi** [xlg8/APMIS-Project#204]
### Feature Description

- [ ] Ability to Import and Export users list.
- [ ] Listing Position after the Organization.

https://afghanistan-apmis.com/sormas-ui/#!users

### Additional Information

![Users](https://user-images.githubusercontent.com/88741672/179012507-8c06f32e-ae6a-436a-84d6-6603132e15cf.JPG)

**Comment by bnygren** [comment]
@ToluOlayinka @Iyanuu This needs further defining of requirements (e.g, about how import could work, password management in bulk, etc.), moved to roadmap

**Comment by ToluOlayinka** [comment]
> @ToluOlayinka @Iyanuu This needs further defining of requirements (e.g, about how import could work, password management in bulk, etc.), moved to roadmap

@bnygren has this been better refined? so we could work on the Issue

**Comment by RaziaMahmodi** [comment]
> > @ToluOlayinka @Iyanuu This needs further defining of requirements (e.g, about how import could work, password management in bulk, etc.), moved to roadmap
> 
> @bnygren has this been better refined? so we could work on the Issue

No, not yet. More information for this issue, task number 1 (first draft of definition) will be available by start of this week. And task number 2 is clear.

**Comment by RaziaMahmodi** [comment]
More information about task number 1 will be available after Sep NID. Thanks.

**Comment by RaziaMahmodi** [comment]
I moved the previous second task into new issue #258

**Comment by ToluOlayinka** [comment]
Based on the last meeting @Iyanuu @RaziaMahmodi we would be working on this to Export Data only.
